agcaballero/portfolio/src/main/webapp/comments.html

```diff
@@ -5,18 +5,21 @@
     <title>Comments</title>
     <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.js"
+      integrity="sha256-2Kok7MbOyxpgUVvAk/HJ2jigOSYS2auK4Pfzbm7uH60="
```

Comment:
Just for my own edification: What is the integrity hash used for?

Reply:
I originally put it in there because the resource I learned how to use jquery from had the script tag with an integrity hash. I think [https://hacks.mozilla.org/2015/09/subresource-integrity-in-firefox-43/](url) this article explains it best